### PR TITLE
issue13 add task is finished

### PR DESCRIPTION
**Pull request by Hailey-cloud**

### CODE REVIEW

File: to_do_list_part_two.py [hunk 1]
```diff
@@ -1,7 +1,16 @@
 def add_task(task_list):
-    add_your_task = input("Enter the task:    ")
-    task_list.append(add_your_task)
-    print(f"{add_your_task} has been added to the list.")
+    task = input("Enter the task: ")
+
+    priority = input("Enter the priority(high, medium, low): ")
+    deadline = input("Enter the dead line(YYYY-MM-DD): ")
```
Comment (shigeokitamura) [inline to_do_list_part_two.py]: In this line, deadline is str type.
We should discuss about the date type of the deadline: just leave it in str type or convert to datetime type

File: to_do_list_part_two.py [hunk 2]
```diff
@@ -1,7 +1,16 @@
 def add_task(task_list):
-    add_your_task = input("Enter the task:    ")
-    task_list.append(add_your_task)
-    print(f"{add_your_task} has been added to the list.")
+    task = input("Enter the task: ")
+
+    priority = input("Enter the priority(high, medium, low): ")
```
Comment (shigeokitamura) [inline to_do_list_part_two.py:14]: You can check whether priority is `high` or `medium` or `low` here.

In line 7, you only check whether priority is empty or not.

File: to_do_list_part_two.py (hunk 1 above)
Comment (shigeokitamura) [reply to_do_list_part_two.py]: We decided to keep deadline as str type.
no need to change.

File: .idea/To-Do-List-Application.iml
Comment (shigeokitamura) [inline .idea/To-Do-List-Application.iml:1]: Can you delete this file from your commit?
This file is not needed for the repository.

**Review by shigeokitamura — CHANGES_REQUESTED** [review]

**Review by shigeokitamura — APPROVED** [review]

File: to_do_list_part_two.py [hunk 3]
```diff
@@ -1,7 +1,21 @@
...
+    if priority != "high" or "medium" or "low":
+
+        print("Enter the correct word in high,medium,low")
+        return
+
+    deadline = input("Enter the dead line(YYYY-MM-DD): ")
```
Comment (shigeokitamura) [inline to_do_list_part_two.py]: Can you handle invalid inputs?
Such as entering an incorrect date format like `2024/10/15`

**Review by kiarash2077 — APPROVED** [review]

**Pull request merged**